stemcell_builder/stages/image_install_grub/apply.sh

```diff
@@ -68,18 +68,25 @@ then
   # GRUB 2 needs /sys and /proc to do its job
   mount -t proc none ${image_mount_point}/proc
   add_on_exit "umount ${image_mount_point}/proc"
-  
+
   mount -t sysfs none ${image_mount_point}/sys
   add_on_exit "umount ${image_mount_point}/sys"
-  
+
   echo "(hd0) ${device}" > ${image_mount_point}/device.map
 
   # install bootsector into disk image file
   run_in_chroot ${image_mount_point} "grub2-install -v --no-floppy --grub-mkdevicemap=/device.map ${device}"
 
-  cat >${image_mount_point}/etc/default/grub <<EOF
+  # Append "console=ttyS0,115200n8 earlyprintk=ttyS0 rootdelay=300" for azure
+  if [ "${stemcell_infrastructure}" == "azure" ]; then
+    cat >${image_mount_point}/etc/default/grub <<EOF
+GRUB_CMDLINE_LINUX="vconsole.keymap=us net.ifnames=0 crashkernel=auto selinux=0 plymouth.enable=0 console=ttyS0,115200n8 earlyprintk=ttyS0 rootdelay=300"
```

Comment:
why do we need rootdelay? what about earlyprintk?

i believe we set console=ttyS0,115200n8 configuration for openstack stemcells. i think we can always configure it for all stemcells.